synapse/lib/storm.py

```diff
@@ -4321,6 +4321,82 @@ async def execStormCmd(self, runt, genr):
                 await self.delEdges(node, verb, n2)
                 yield node, path
 
+class OnceCmd(Cmd):
+    '''
+    The once command ensures that a node makes it through the once command but a single time,
+    even across independent queries. The gating is keyed by a required name parameter to
+    the once command, so a node can be run through different queries, each a single time, so
+    long as the names differ.
+
+    For example, update the seen times on a set of files just once:
+
+        file:bytes#my.files | once node:seen | [ +.seen=$lib.time.now() ]
```

Comment:
The example using a `node:seen` property and updating the `.seen` property only once is a bit counter intuitive.  Could we change to something like `enrich:foo` and ` | foo.enrich` to prevent any confusion about the use case?